Sources/StreamChat/Models/Attachments/AttachmentTypes.swift

```diff
@@ -292,6 +294,15 @@ public enum AttachmentFileType: String, Codable, Equatable, CaseIterable {
             .first(where: { $1 == self })?
             .key ?? "application/octet-stream"
     }
+
+    public var isAudio: Bool {
+        switch self {
+        case .mp3, .mp4, .wav, .ogg, .m4a, .aac:
```

Comment:
Is mp4 an audio? 🤔 Mp4 is a container format, and usually, it contains both video and audio codecs. This could be misleading, if someone will use this property to render an audio, but in fact, it is a video 🤔 Where is this property being used?

Reply:
https://github.com/GetStream/stream-chat-swift/pull/2639